pmm-app/src/pmm-qan/panel/QueryAnalytics.constants.test.tsx

```diff
@@ -0,0 +1,552 @@
+
+    expect(processedMetric).toBe('');
+  });
+
+  it('query_time', () => {
+    // Metric with empty relation
+    const metric = METRIC_CATALOGUE.query_time;
+    const processedMetric = metric.metricRelation();
+
+    expect(processedMetric).toBe('');
+  });
+
+  it('response_length', () => {
+    const metric = METRIC_CATALOGUE.response_length;
+    const processedMetric = metric.metricRelation(metrics);
+    const absentMetric = metric.metricRelation({});
+
+    expect(processedMetric).toBe('1.00 Bytes bytes/row');
+    expect(absentMetric).toBe('');
+  });
+
+  it('rows_affected', () => {
+    // Metric with empty relation
+    const metric = METRIC_CATALOGUE.rows_affected;
+    const processedMetric = metric.metricRelation();
+
+    expect(processedMetric).toBe('');
+  });
+
+  it('rows_examined', () => {
+    const metric = METRIC_CATALOGUE.rows_examined;
+    const processedMetric = metric.metricRelation(metrics);
+    const absentMetric = metric.metricRelation({});
+
+    expect(processedMetric).toBe('1.00 per row sent');
+    expect(absentMetric).toBe('');
+  });
+
+  it('rows_read', () => {
+    // Metric with empty relation
+    const metric = METRIC_CATALOGUE.rows_read;
+    const processedMetric = metric.metricRelation();
+
+    expect(processedMetric).toBe('');
+  });
+
+  it('rows_sent', () => {
+    // Metric with empty relation
+    const metric = METRIC_CATALOGUE.rows_sent;
+    const processedMetric = metric.metricRelation();
+
+    expect(processedMetric).toBe('');
+  });
+
+  it('select_full_range_join', () => {
+    // Metric with empty relation
+    const metric = METRIC_CATALOGUE.select_full_range_join;
+    const processedMetric = metric.metricRelation();
+
+    expect(processedMetric).toBe('');
+  });
+
+  it('sort_range', () => {
+    // Metric with empty relation
+    const metric = METRIC_CATALOGUE.sort_range;
+    const processedMetric = metric.metricRelation();
+
+    expect(processedMetric).toBe('');
+  });
+
+  it('select_range', () => {
+    // Metric with empty relation
+    const metric = METRIC_CATALOGUE.select_range;
+    const processedMetric = metric.metricRelation();
+
+    expect(processedMetric).toBe('');
+  });
+
+  it('select_range_check', () => {
+    // Metric with empty relation
+    const metric = METRIC_CATALOGUE.select_range_check;
+    const processedMetric = metric.metricRelation();
+
+    expect(processedMetric).toBe('');
+  });
+
+  it('sort_rows', () => {
+    // Metric with empty relation
+    const metric = METRIC_CATALOGUE.sort_rows;
+    const processedMetric = metric.metricRelation();
+
+    expect(processedMetric).toBe('');
+  });
+
+  it('sort_scan', () => {
+    // Metric with empty relation
+    const metric = METRIC_CATALOGUE.sort_scan;
+    const processedMetric = metric.metricRelation();
+
+    expect(processedMetric).toBe('');
+  });
+
+  it('tmp_disk_tables', () => {
+    // TODO: check correct values, not sure about units
```

Comment:
Can't this TODO's be addressed in the current task? What units are missing? 🤔

Reply:
It will require some investigation so I created a separate ticket for it, almost sure that it works ok now, only want to be sure 100%